lib/command_cursor.js

```diff
@@ -140,116 +167,6 @@ Object.defineProperty(CommandCursor.prototype, 'namespace', {
  * @type {null}
  */
 
-// Inherit from Readable
-inherits(CommandCursor, Readable);
-
-// Set the methods to inherit from prototype
-var methodsToInherit = [
```

Comment:
So what happened to all the methods that we were explicitly not inheriting. Are we inheriting them now?

Reply:
That's correct, however it's unclear to me whether that's a bad thing. For starters, this list includes items that don't exist (like `_find`), exemplifying the brittle nature of this approach. Here is a list of methods the `CommandCursor` gains:
```
> now.filter(x => methodsToInherit.indexOf(x) < 0)
[ 'constructor',
  'readPreference',
  '_initializeCursor',
  'filter',
  'maxScan',
  'hint',
  'min',
  'max',
  'returnKey',
  'showRecordId',
  'snapshot',
  'setCursorOption',
  'addCursorFlag',
  'addQueryModifier',
  'comment',
  'maxAwaitTimeMS',
  'maxTimeMS',
  'project',
  'sort',
  'batchSize',
  'collation',
  'limit',
  'skip',
  'setReadPreference',
  'count',
  'map',
  'destroy',
  'stream',
  'transformStream',
  'getLogger',
  'maxTimeMs' ]
```